### ISSUE TITLE
Force graph visualization

### ISSUE THREAD

**Issue by Dev1an** [Quite-nice/Visualisation#11]
Visualize realtime messages in a force graph. https://github.com/d3/d3/wiki/Force-Layout

**Comment by maccradar** [comment]
I would immediately add a useful feature: display bottlenecks. the force graph on its own is neat but does not do anything useful, except for displaying the connections (which in zyre is a complete mesh).

If you add a feature which highlights connections or nodes that process a lot of messages, we could quickly identify potential bottlenecks. See also #20

**Comment by Dev1an** [comment]
Work in progress in https://github.com/Quite-nice/Forcegraph

**Comment by maccradar** [comment]
Can I already link it to a collection which grabs zyre messages out of the database?

**Comment by Dev1an** [comment]
Yes the forcegraph is coupled to a Modules and Events collection, so if you run the zyre mediator alongside the meteor app you can see the zyre messages. I am now integrating this project in the Visualization project. When that is done, I will add an indicator for the _sent_ and _received message rate_ to identify bottlenecks.

**Comment by Dev1an** [comment]
In order to integrate this, I was waiting on the solution for #22.
